app/navigation/NavigatorForDisclosure.scala

```diff
@@ -19,7 +19,7 @@ package navigation
 import models.Mode
 import models.disclosure.DisclosureType.{Dac6add, Dac6new}
 import pages.Page
-import pages.disclosure.{DisclosureMarketablePage, DisclosureNamePage, DisclosureTypePage}
+import pages.disclosure._
 import play.api.mvc.Call
 
 object NavigatorForDisclosure extends AbstractNavigator {
```

Comment:
Can you switch to an injectable version of the navigator? I know I suggested them to be static, but during the test for DAC6-464 I realized the injected versions work better. Please, look here: https://github.com/hmrc/enter-cross-border-arrangements-frontend/blob/88bf0ec2e3471f37fa0e615c5651994eab407e2e/app/navigation/NavigatorForIndividual.scala